src/shared/views/Mapper/Tile.js

```diff
@@ -143,8 +151,14 @@ export class _Tile extends React.Component<Props> {
         return { uri: tile.url };
     }
 
+    getOsmBuildings = () => {
+        const { tile } = this.props;
+        return { uri: tile.urlB };
```

Comment:
I suspect `yarn flow` is going to complain about this new attribute on the `tile` object, you will probably need to update the type definitions.